packages/app/src/app/overmind/internalActions.ts

```diff
@@ -469,6 +469,7 @@ export const identifyCurrentUser: AsyncAction = async ({ state, effects }) => {
   const user = state.user;
   if (user) {
     effects.analytics.identify('pilot', user.experiments.inPilot);
+    effects.browser.storage.set('pilot', user.experiments.inPilot);
```

Comment:
Shouldn't there be a switch in the Experiments modal to enable this? 🤔

Reply:
We only enable the teams that we invite currently, so it's stored in the database. We only store it in the localStorage so we won't have to wait for an API call to finish before redirecting:

https://github.com/codesandbox/codesandbox-client/blob/add/pilot-new-dashboard/packages/app/src/app/pages/Dashboard/Content/index.js#L20-L22